src/hair/hairStyle.css

```diff
@@ -0,0 +1,144 @@
+@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:400,700&display=swap');
+
+body {
+  height: 100vh;
+  width: 100vw;
+  background-image: url(./img/mainbackgroundimg.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  /* how do i know when to do vh vs %? */
+  display: grid;
+  grid-template-columns:200px repeat(5, 2fr);
+  grid-template-rows: 200px repeat(9, 1fr);
+  font-family: 'Josefin Sans', sans-serif !important;
+}
+
+aside {
+  grid-area: 1/1/-1/2;
+  display: grid;
+  grid-template-columns: 200px;
+  grid-template-rows: inherit;
+  /* background-color: rgb(73, 72, 71); 
+  here i learned cant have bg-color & bkg rgba regardless of order*/
+  background:rgba(251,249,244,0.9);
```

Comment:
oh yeah, background is short for all the other background properties ( color, size, position...) you can use it when you want to put them all on one line. [background property](https://www.w3schools.com/cssref/css3_pr_background.asp)